pkg/logs/logs.go

```diff
@@ -153,3 +158,7 @@ func NewContext(ctx context.Context, l logr.Logger, names ...string) context.Con
 	}
 	return context.WithValue(ctx, contextKey, l)
 }
+
+func V(level int) klog.Verbose {
+	return klog.V(klog.Level(level))
+}
```

Comment:
If we changed the signature to expect a type we could ensure that only named log levels are used....I think.

```suggestion
func V(level LogLevel) klog.Verbose {
	return klog.V(klog.Level(level))
}
```

Reply:
I tried this, this breaks the logr interface :(